base58.py

```diff
@@ -27,9 +27,20 @@
     buffer = lambda s: s.buffer
 
 
+def int_to_b58(i):
```

Comment:
I would prefer to have these names symmetrical with the ones already existing, so something like `b58encode_int` and `b68decode_int`